nautobot/extras/plugins/__init__.py

```diff
@@ -35,6 +35,7 @@
 registry["plugin_graphql_types"] = []
 registry["plugin_jobs"] = []
 registry["plugin_template_extensions"] = collections.defaultdict(list)
+registry["plugin_metrics"] = []
```

Comment:
Since we're trying to rename `"plugins"` to `"apps"` in the near term, let's use `app_metrics` throughout here even though it's less self-consistent for now.

Reply:
Done.